### ISSUE TITLE
Pi-Hole / No Internet when open an Wifi / Tor is not starting

### ISSUE THREAD

**Issue by ghost** [webnull/RaspAP#12]
Hello, at first its a great work i like it thank you!
I have install RaspAP and i have start a Hotspot with a diffrent ip as eth0 i can connect but there is no Internet, is it a issu from me?
Next i tried to insatll also pi-hole but i dont can open the Webinterface from Pi-hole becouse both are on the same port? Can i with the port or can you build a selection which port i like to use?

I also tried bevor i istall pi-hole to start tor as Proxy i fill in like:
VirtualAddrNetworkIPv4 10.192.0.0/10
AutomapHostsSuffixes .onion,.exit
AutomapHostsOnResolve 1
TransPort 192.168.42.1:9040
DNSPort 192.168.42.1:9053

when i like to save the settings nothing happened....

Thank You :-)

**Comment by blackandred** [comment]
I think you have to run also the daemon?

There is a web gui + daemon.

**Comment by ghost** [comment]
yes tor is running and the daemon i only can change teh configuration of tor per ssh not in the Raspap web-GUI.
But so far Tor is working my next problem is when i configure eth1 and wlan0 with the same IP and DHCP Range Tor only works over eth1, when i disconnect eth1 wlan0 is working with tor but both at the same time should not be possible.. have you any idea?

**Comment by ghost** [comment]
I also like to upgrade Raspap from 1.4.0 to 1.4.1 is there a way to upgrade withou loosing my configuration?

**Comment by blackandred** [comment]
I think Raspap in line 1.x is maintained under other repository, you can ask there: https://github.com/billz/raspap-webgui/issues :wink:

**Comment by ghost** [comment]
okay thx i will try there my luck :-) Thank you
